_pages/integrations.md

```diff
@@ -1,33 +1,34 @@
 ---
 title: Third-Party Integrations
 ---
+
 # Third-Party Integrations
 
 Our software is built on the shoulders of giants. We use several third parties
 to perform continuous monitoring, host our apps, etc.; all should be approved
-on the [IT Standards
-list](https://github.com/GSA/data/blob/gh-pages/enterprise-architecture/it-standards.csv).
+on the [GSA IT Standards
+list](https://ea.gsa.gov/#!/itstandards). A public, but only periodically updated, [snapshot of the IT Standards](https://github.com/GSA/data/blob/master/enterprise-architecture/it-standards.csv) is also available.
```

Comment:
I believe ea.gsa.gov is behind the VPN, might want to mark this as such?

Reply:
Oh yeah – I had briefly looked into how the TTS handbook on how we do that, but forgot to open an issue – just opened #150. It's done by https://github.com/18F/private-eye. 

Once that plug-in is added, that URL will get the standard lock/VPN-only icon without any changes to this markup.

Reply:
Perfect! :raised_hands: